MySQL/system/ksp-block-stig-mysql-files-access.yaml

```diff
@@ -0,0 +1,39 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-stig-mysql-files-access
+  namespace: default # Change your namespace
+spec:
+  tags: ["STIG", "MySQL"]
+  message: "Alert! Access to MySQL conf files has been denied"
+  selector:
+    matchLabels:
+      pod: mysql-1 # Change your match labels
+  file:
+    severity: 5
+    matchPaths:
+    - path: /etc/mysql/my.cnf
+      ownerOnly: true
+    - path: /etc/mysql/mariadb.conf.d
+      ownerOnly: true
+    - path: /etc/mysql/conf.d
+      ownerOnly: true
+    - path: /etc/mysql/debian-start
+      ownerOnly: true
+    - path: /etc/mysql/debian.cnf
+      ownerOnly: true
+    - path: /etc/mysql/mariadb.cnf
+      ownerOnly: true
+    - path: /mysqld.pid
+      ownerOnly: true
+    - path: /tmp/mysql.sock
+      ownerOnly: true
+    - path: /usr/lib/systemd/system/mysql.service
+      ownerOnly: true
+    - path: /usr/lib/systemd/system/mysqld.service
+      ownerOnly: true
+    action: Block
```

Comment:
This policy does not makes sense. If we block access to these configuration files, the workload is not going to operate.